src/app/gallery/gallery.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="g-screen-container flex flex-col gap-5">
   <div>
-    <div class="g-title-1">Gallery</div>
-    <div class="g-sub-title">Explore community prompt</div>
+    <div class="g-title-1-class">Gallery</div>
+    <div class="g-sub-title-class">Explore community prompt</div>
```

Comment:
g-sub-title was already a project specific class (and not a tailwind one), no need to add '-class' at the end.